app/src/main/java/org/simple/clinic/searchresultsview/PatientSearchView.kt

```diff
@@ -47,6 +48,7 @@ class PatientSearchView(context: Context, attrs: AttributeSet) : RelativeLayout(
   private val emptyStateView by bindView<View>(R.id.searchresults_empty_state)
   private val newPatientRationaleTextView by bindView<TextView>(R.id.searchresults_new_patient_rationale)
   private val newPatientButton by bindView<Button>(R.id.searchresults_new_patient)
+  private val loader by bindView<ProgressBar>(R.id.searchresults_loader)
```

Comment:
Switch to the Kotlin view extensions? We have been moving screens to use them as and when we need to change them.

Reply:
Done.